src/EnergyPlus/ConvectionCoefficients.cc

```diff
@@ -280,6 +280,7 @@ void InitIntConvCoeff(EnergyPlusData &state,
         auto const &zone = state.dataHeatBal->Zone(ZoneNum);
         for (int spaceNum : zone.spaceIndexes) {
             auto const &thisSpace = state.dataHeatBal->space(spaceNum);
+            Real64 spaceMAT = state.dataZoneTempPredictorCorrector->spaceHeatBalance(spaceNum).MAT;
```

Comment:
Small change to make this a bit more readable.  It's not a functional change, is it?  I can't tell from here if that argument is marked INTENT(INOUT) -- I mean as a reference :)

Reply:
Key thing here is changing from zone MAT to space MAT.